auto_nag/scripts/duplicate_copy_metadata.py

```diff
@@ -263,11 +276,26 @@ def get_previously_added_regressors(self, bug: dict) -> Set[int]:
 
         return added_regressors
 
+    def was_confirmed(self, bug: dict) -> bool:
+        """Check if the bug was confirmed."""
+
+        for entry in reversed(bug["history"]):
+            for change in entry["changes"]:
+                if change["field_name"] == "status" and change["removed"] not in (
+                    "REOPENED",
+                    "CLOSED",
+                    "RESOLVED",
+                ):
+                    return change["removed"] != "UNCONFIRMED"
```

Comment:
Shouldn't we return True when the bug was UNCONFIRMED in the past? So shouldn't this be `change["removed"] == "UNCONFIRMED"`?

Reply:
The final status for a duplicate bug is `RESOLVED`; the status before that should not be `UNCONFIRMED`.

We will consider the bug confirmed if the removed status is `NEW`, `ASSIGNED`,  or `VERIFIED`. However, we will keep looking for the previous changes when the removed status is `REOPENED`,  `CLOSED`, or `RESOLVED`.

Reply:
Sorry I was reading this backwards!